core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.template.html

```diff
@@ -22,14 +22,14 @@ <h3>
       <div class="story-input-header">
         <span>Description*</span>
       </div>
-      <textarea rows="3" cols="50" ng-model="story.description" class="form-control protractor-test-new-story-description-field" aria-label="Description"></textarea>
+      <textarea rows="3" cols="50" ng-model="story.description" class="form-control protractor-test-new-story-description-field required" aria-label="Description"></textarea>
```

Comment:
Hey @kevintab95 Having a class named required can be confusing with bootstrap. Would oppia-required-input or something would be better?